src/cf-layout/src/molecules/heroes.less

```diff
@@ -120,14 +120,14 @@
             }
 
             .m-hero_image {
-                // Remove the flexible container padding so that the image is a consistent width
-                padding-bottom: 0 !important;
+                
+                padding-bottom: 0;
                 height: 100%;
                 width: 100%;
                 position: absolute;
                 background-size: cover;
             }
-        } );
+        } ) !important;
```

Comment:
Are we sure we want all of these things within the media query to have `!important`?